libtransmission/bandwidth.cc

```diff
@@ -23,50 +23,49 @@
 ****
 ***/
 
-unsigned int Bandwidth::getSpeed_Bps(RateControl const* r, unsigned int interval_msec, uint64_t now)
+unsigned int Bandwidth::getSpeedBytesPerSecond(RateControl& r, unsigned int interval_msec, uint64_t now)
```

Comment:
Later when we introduce data rate units, this should become getSpeed or even getDataRate